app/models/manageiq/providers/amazon/cloud_manager/refresh_parser_inventory_object.rb

```diff
@@ -143,6 +147,7 @@ def parse_flavor(flavor)
 
     {
       :type                     => ManageIQ::Providers::Amazon::CloudManager::Flavor.name,
+      :ext_management_system    => ems,
```

Comment:
The ems relation is usually filled automatically, when it's saved like ems.association.build, but for a targeted refresh, we don't build it like this, so we need to provide it.